From 0b9fb976e987913a38a05397ef4df6801842e7e9 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 2 Oct 2023 17:12:11 +0200
Subject: [PATCH 363/954] fix(stm32mp2): correct typo in definition header

Fix a typo about the platform in a comment (STM32MP2 instead
of STM32MP1).

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 8b67ed631a6689093a54332a639c55f87797a842 fix(stm32mp2): correct typo in definition header]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I6a58b659d97d7143e277dea57d4eede7729092bc
---
 plat/st/stm32mp2/stm32mp2_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index 206703e895..695b9144c0 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -447,7 +447,7 @@ static inline uintptr_t tamp_bkpr(uint32_t idx)
 #define IWDG2_BASE				U(0x44020000)
 
 /*******************************************************************************
- * Miscellaneous STM32MP1 peripherals base address
+ * Miscellaneous STM32MP2 peripherals base address
  ******************************************************************************/
 #define BSEC_BASE				U(0x44000000)
 #define DBGMCU_BASE				U(0x4A010000)
-- 
2.53.0

